standalone-metastore/metastore-rest-catalog/pom.xml

```diff
@@ -43,6 +43,12 @@
       <version>${hive.version}</version>
     </dependency>
     <!-- Test dependencies -->
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-exec</artifactId>
```

Comment:
why do we need ql dependency here? HMS is supposed to be standalone project

Reply:
could we use core classifier?

Reply:
It likely works!
https://github.com/apache/hive/pull/6171/commits/46810c28c925968ffb39e00e50ce36673a769c43